config/rbac/role.yaml

```diff
@@ -109,6 +109,20 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
This role addition needs to be updated manually in `config/hub/rbac/role.yaml` file as well

Reply:
Added!  It seems `config/rbac/roles.yaml` is a union of `config/hub/rbac/roles.yaml` and `config/dr_cluster/rbac/roles.yaml`.  I wonder why it must be maintained?  If a cluster were both a `hub` and `dr-cluster`, perhaps both could be applied. Perhaps through a `kustomization.yaml` if a second applied would otherwise replace a first.

Reply:
It is just present as generate updates the same  (or possibly would create on new) whenever we use the kubebuilder markers.